src/card.css

```diff
@@ -0,0 +1,77 @@
+.card {
```

Comment:
let card have its paddings – you won't have to give its childs margins

Reply:
The horizontal left and right gaps between photo and card is 19px, but for rest of the elements it is 16px, therefore I couldn't give a card fixed padding

Reply:
thus – add paddings for `card__product`

Reply:
Hmm alright, great idea!